app/js/components/manageApps/ManageApps.jsx

```diff
@@ -158,7 +158,7 @@ export default class ManageApps extends React.Component {
       this.requestUrl = '/v1/module/?v=full';
       axios.get(`${urlPrefix}/${apiBaseUrl}${this.requestUrl}`).then(response => {
         return Promise
-          .all(response.data.results.map(data => axios.get(`https://addons.openmrs.org/api/v1/addon/${data.packageName}`)
```

Comment:
Is addonUrl is property or a method that gets the url?
In other words, if a method, isn't getAddonUrl better?

Reply:
Yes its clearer, i have updated the PR